### DISCUSSION TITLE
Allow custom DB Port

### DISCUSSION THREAD

**Discussion by joshcadman-easyreply** [postalserver/postal#3064] [Feature suggestions]
I have had a look at all available config options and cannot seem to find an option to change a custom port. Some managed Database providers open a random port instead of 3306, so cannot be setup with Postal. Is this something that is likely to be changed within Postal in the future?

**Comment by willpower232** [comment]
For future people who may be confused, I presume you found the complete database settings to set a custom port?

https://github.com/postalserver/postal/blob/da90e75036c27482699921613d838f4058a100e7/doc/config/yaml.yml#L53-L81
